test-cases/terraform/aws/best-practices/using_public_amis/results_summary.json

```diff
@@ -1,7 +1,7 @@
 {
   "checkov": "no",
   "cloudrail": "yes",
-  "kics": "no",
+  "kics": "yes",
```

Comment:
@kaplanlior I'm not seeing where KICS catches the use of public AMI? Can you point me to it?